components/arm/pb_helpers.go

```diff
@@ -4,11 +4,45 @@ import (
 	"fmt"
 
 	commonpb "go.viam.com/api/common/v1"
+	pb "go.viam.com/api/component/arm/v1"
 
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/referenceframe/urdf"
+	"go.viam.com/rdk/utils"
 )
 
+// MoveOptions define parameters to be obeyed during arm movement.
+type MoveOptions struct {
+	MaxVel, MaxAcc float64
```

Comment:
+1

Reply:
Sorry I should've thought of this when I left my original comment, thoughts on the units actually being in degrees since that are easier to mentally reason about? We could then handle the conversion in the backend.
If not, that's also ok :)

Reply:
No the units should be in radians because this is the struct we will be operating on.  The protobuf struct is in degrees